.github/workflows/build-subsystems.yaml

```diff
@@ -82,6 +82,14 @@ jobs:
       dry-run: ${{ inputs.dry-run }}
       is-dependabot: ${{ inputs.is-dependabot }}
 
+  build-openapi:
+    needs: test
+    uses: ./.github/workflows/build-openapi.yaml
+    secrets: inherit
+    with:
+      dry-run: ${{ inputs.dry-run }}
```

Comment:
You have dry run feature toggles enabled, for this. Should it have this?

Reply:
I dont see why not